packages/react/src/components/accordion/accordion-container.test.tsx

```diff
@@ -0,0 +1,191 @@
+import React from 'react';
+import { mountWithTheme } from '../../test-utils/renderer';
+import { AccordionContainer, Accordion } from './index';
+
+describe('AccordionContianer', () => {
+    it('Should have the last accordion opne by default', () => {
+        const wrapper = mountWithTheme(
+            <AccordionContainer defaultExpandedItemIds={['3']}>
+                <Accordion title='Panel 1' id='1'>
+                    Content for Panel 1
+                </Accordion>
+                <Accordion title='Panel 2' id='2'>
+                    Content for Panel 2
+                </Accordion>
+                <Accordion title='Panel 3' id='3'>
+                    Content for Panel 3
+                </Accordion>
+            </AccordionContainer>,
+        );
+
+        const button = wrapper.find('.accordion-button').last();
+        const container = wrapper.find('.accordion-content').last();
+        expect(button.prop('aria-expanded')).toBe(true);
+        expect(container.prop('aria-expanded')).toBe(true);
+    });
+
+    it('Should have the First and Last accordion opne by default', () => {
+        const wrapper = mountWithTheme(
+            <AccordionContainer defaultExpandedItemIds={['1', '3']}>
+                <Accordion title='Panel 1' id='1'>
+                    Content for Panel 1
+                </Accordion>
+                <Accordion title='Panel 2' id='2'>
+                    Content for Panel 2
+                </Accordion>
+                <Accordion title='Panel 3' id='3'>
+                    Content for Panel 3
+                </Accordion>
+            </AccordionContainer>,
+        );
+
+        const firstButton = wrapper.find('.accordion-button').first();
+        const firstContainer = wrapper.find('.accordion-content').first();
+        const lastButton = wrapper.find('.accordion-button').last();
+        const LastContainer = wrapper.find('.accordion-content').last();
+        expect(firstButton.prop('aria-expanded')).toBe(true);
+        expect(firstContainer.prop('aria-expanded')).toBe(true);
+        expect(lastButton.prop('aria-expanded')).toBe(true);
+        expect(LastContainer.prop('aria-expanded')).toBe(true);
+    });
+
+    it('Should have the First accordion disabled by default', () => {
+        const wrapper = mountWithTheme(
+            <AccordionContainer disabledItemIds={['1']}>
```

Comment:
it is an extra feature. Just in case one have the limitation on adding disabled.

Reply:
I agree with @savutsang here. I believe it's more straightforward to specify the state of the accordions indivudally than through a `disabledItemIds` prop. It would also expose an artifical concept of ids. There are a few things we need to consider to keep the state coherent, but I thin it gives a better API for the component.

There's also the option of providing Accordions elements through an object array that is converted in React elements in the  Accordion component instead of providing ReactElements with the `children` prop.